s3upload.js

```diff
@@ -59,7 +59,7 @@ S3Upload.prototype.createCORSRequest = function(method, url) {
 
 S3Upload.prototype.executeOnSignedUrl = function(file, callback) {
     var xhr = new XMLHttpRequest();
-    var fileName = file.name.replace(/[^\w]/g, "_");
+    var fileName = file.name.replace(/\ /g, "_");
```

Comment:
Not sure you need to escape a space in your regex, so probably could just be `/ /g`, but maybe let's just use `/\s+/g` to replace all whitespace with `_`